misc/m.transform/m.transform.html

```diff
@@ -53,12 +53,14 @@ <h2>NOTES</h2>
 </pre></div>
 
 ["." = dot-product, (AE).N = N'EA.]
```

Comment:
I have changed it to `&middot;` (inspired by https://stackoverflow.com/a/37965943/452464) in f8ea2004d2b8b06c9293d41fbe27d6aee2c12044

Please verify, as I am not sure about the last dot in `["·" = dot-product, (AE)·N = N'EA.]`

Reply:
I'm not sure about the last dot too. Seems like a sentence ending as it doesn't make sense as math (from all my math knowledge, I don't see what it could be)

Reply:
I have removed this apparently superfluous dot in ea808b8a205e5f152244fcb7015f33a3cfd09f3e